abb_resources/urdf/common_colours.xacro

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://ros.org/wiki/xacro">
+  <!-- Conversion were obtained from http://www.e-paint.co.uk/Lab_values.asp
+       unless otherwise stated. -->
+
+  <!-- Actual colors -->
+  <!-- NCS 2070-Y60R orange, standard robot color 1974 - May 2014 -->
+  <xacro:property name="colour_abb_orange" value="0.7372549 0.3490196 0.1607843 1" />
+
+  <!-- RAL 7035 light grey (abb graphite white), standard robot color May 2014 - current -->
+  <xacro:property name="colour_abb_graphite_white" value="0.7725490 0.7803922 0.7686275 1" />
+
+  <!-- RAL 9003 signal white (abb white), cleanroom robot color -->
+  <xacro:property name="colour_abb_white" value="0.9254902 0.9254902 0.9058824 1" />
+
+  <!-- RAL 7032 pebble gray, controller -->
+  <xacro:property name="colour_abb_pebble_gray" value="0.7058823 0.6901961 0.6313725 1" />
+
+  <!-- RAL 7012 basalt gray (abb dark gray) , robot with dark link color -->
+  <xacro:property name="colour_abb_dark_gray" value="0.3450980 0.3647059 0.3686275 1" />
+
+  <!-- Other robot colours availble, but not limited too. -->
+  <!-- RAL 3001 signal red -->
+  <xacro:property name="colour_abb_signal_red" value="0.5960784	0.1372549 0.1372549 1" />
```

Comment:
Not sure why it is showing up that way, but if you view the file it is not present.